Application/Telegram/MainActions/NewBookingCommand.cs

```diff
@@ -105,7 +105,7 @@ public static async Task HandleAsync(Update update, TelegramBotClient telegraBot
                         , new List<string> { "Standart Booking", "Booking By Workpalce Attribute", "BACK" }
                         , "Please select workplace booking type"
                         , 2
-                        , "BookingIsSelected");
+                        , "NewBookingIsSelected");
```

Comment:
why are you touching 'NewBookingCommand' ? you need to implement it in 'NewVacationCommand'

Reply:
Because when I click BACK button it did not worked then I found the issue and fixed it to NewBookingIsSelected. We have no BookingIsSelected state

Reply:
ok